Source/FieldSolver/FiniteDifferenceSolver/EvolveE.cpp

```diff
@@ -49,11 +49,21 @@ void FiniteDifferenceSolver::EvolveE (
     std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& Bfield,
     std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& Jfield,
     std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& edge_lengths,
+    std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& face_areas,
+    std::array< std::unique_ptr<amrex::MultiFab>, 3 >& Rhofield,
     std::unique_ptr<amrex::MultiFab> const& Ffield,
     int lev, amrex::Real const dt ) {
 
-   // Select algorithm (The choice of algorithm is a runtime option,
-   // but we compile code for each algorithm, using templates)
+#ifdef AMREX_USE_EB
+    if (m_fdtd_algo != MaxwellSolverAlgo::ECT) {
+        amrex::ignore_unused(face_areas, Rhofield);
+    }
+#else
+    amrex::ignore_unused(face_areas, Rhofield);
+#endif
```

Comment:
My understanding is that `amrex::ignore_unused` only affects the compile-time behavior. Therefore, I don't think that the first part of that `#ifdef` does anything (since it depends on a run-time condition).
```suggestion
#ifndef AMREX_USE_EB
    amrex::ignore_unused(face_areas, Rhofield);
#endif
```